cypress/integration/Pagination.spec.ts

```diff
@@ -1,293 +1,363 @@
 import * as h from '../helpers';
 
-describe('Button', () => {
+function getPaginationNav() {
+  return cy.findByLabelText('Pagination');
+}
```

Comment:
I'm a huge ball of meh on these types of functions. I personally don't use them as they create an abstraction from the Command Log in a Cypress test making it a bit harder to match up the line that failed. There are 2 problems with this:
1. Command log doesn't match what's in a test. The command log shows `findByLabelText Pagination` while the test shows a `getPaginationNav` function
2. Cypress shows the exact line that fails in a test. It will show this line rather than inside an `it` or `beforeEach` function call

These are definitely not deal breakers, but do aid to slow down the debug process. The only upside of this abstraction is reducing duplication which is not a concern I have in test files.

Reply:
There have been times I've needed to update spec code because a label changed or something. A find/replace still worked.

Reply:
Updated.